.github/workflows/stale-issues.yml

```diff
@@ -0,0 +1,26 @@
+name: "Close stale issues"
+on:
+  schedule:
+    - cron: "0 0 * * *"          # once a day at midnight
+
+permissions: {}
+jobs:
+  stale:
+    permissions:
+      issues: write              # to close stale issues (actions/stale)
+      pull-requests: write       # to close stale PRs (actions/stale)
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue is marked stale. It will be closed in 30 days if it is not updated.'
```

Comment:
What about a refined message sequence like:

Notification about to close:

```
If you would like us to look at this issue, please provide the requested information. If the information is not provided within the next … days this issue will be closed.
```

Close message:

```
Closing due to lack of requested feedback. If you would like us to look at this issue, please provide the requested information and we will re-open the issue.
```

Reply:
Sure, sounds better. Changing this.